packages/core/src/Modules/Endpoint/__tests__/FeatureFlagsSection.spec.tsx

```diff
@@ -95,16 +106,14 @@ describe('<FeatureFlagsSection/>', () => {
         location.hostname = 'staging-app.deriv.com';
         render(mockFeatureFlagsSection());
 
-        expect(screen.getByRole('checkbox', { name: FLAGS.NEXT_WALLET })).toBeInTheDocument();
         expect(screen.getByRole('checkbox', { name: FLAGS.SHARKFIN })).toBeInTheDocument();
         expect(screen.getByRole('checkbox', { name: FLAGS.DTRADER_V2 })).toBeInTheDocument();
         expect(screen.queryByRole('checkbox', { name: FLAGS.WALLET })).not.toBeInTheDocument();
     });
-    it('should render a single next_wallet flag on production', () => {
+    it('should render none of the flags on production', () => {
         location.hostname = website_domain;
         render(mockFeatureFlagsSection());
 
-        expect(screen.getByRole('checkbox', { name: FLAGS.NEXT_WALLET })).toBeInTheDocument();
```

Comment:
small nit (non-blocking) but wouldn't it be better if we check for
`expect(screen.queryByRole('checkbox', { name: FLAGS.NEXT_WALLET })).not.toBeInTheDocument();` instead?

Reply:
But there's no more `NEXT_WALLET` in `FLAGS`, and in general the intended result of the PR is no occurrences of `"next_wallet"` (case-insensitive) in the project code